packages/go/cypher/frontend/expression.go

```diff
@@ -74,12 +74,6 @@ func NewExpressionVisitor() Visitor {
 	return &ExpressionVisitor{}
 }
 
-func (s *ExpressionVisitor) EnterOC_Expression(ctx *parser.OC_ExpressionContext) {
```

Comment:
All EnterOC_* and ExitOC_* are already defined as empty functions on BaseVisitor, so it's safe to remove the empty functions in other visitors.